<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementLibraryDialog</class>
 <widget class="QDialog" name="ElementLibraryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Графические элементы</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Простые</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="simpleGridLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Библиотеки</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,0">
       <item row="0" column="0">
        <layout class="QGridLayout" name="libraryGridLayout"/>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbLoadLibrary">
           <property name="text">
            <string>Загрузить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
